test/broadcasting.jl

```diff
@@ -0,0 +1,364 @@
+module TestBroadcasting
+
+using Test, DataFrames
+
+refdf = DataFrame(ones(3, 5))
+
+@testset "normal data frame and data frame row in broadcasted assignment - one column" begin
+    df = copy(refdf)
+    df[1] .+= 1
+    @test df.x1 == [2, 2, 2]
+    @test df[2:end] == refdf[2:end]
+
+    dfv = @view df[1:2, 2:end]
+    dfv[1] .+= 1
+    @test dfv.x2 == [2, 2]
+    @test dfv[2:end] == refdf[1:2, 3:end]
+    @test all(Matrix(df[1:2, 1:2]) .== 2)
```

Comment:
Also test that `df` is correct? Same in other places when using views and `DataFrameRow`.

Reply:
done - now I always check the whole `df` against what is expected.